hsds/chunk_sn.py

```diff
@@ -924,15 +958,14 @@ async def PUT_Value(request):
             crawler_status = crawler.get_status()
 
             if crawler_status not in (200, 201):
-                log.warn(
-                    f"crawler failed for page: {page_number+1} with status: {crawler_status}"
-                )
+                msg = f"crawler failed for page: {page_number+1} with status: {crawler_status}"
+                log.warn(msg)
             else:
                 log.info("crawler write_chunk_hyperslab successful")
 
     else:
         #
-        # Do point PUT
+        # Do point post
```

Comment:
Other log messages statements here indicate this is a PUT request. If this is a POST request when done for points, I think the following log messages should be updated as well.

Reply:
updated